CoreScriptsRoot/Modules/Settings/Pages/GameSettings.lua

```diff
@@ -53,6 +54,7 @@ local utility = require(RobloxGui.Modules.Settings.Utility)
 RobloxGui:WaitForChild("Modules"):WaitForChild("TenFootInterface")
 RobloxGui:WaitForChild("Modules"):WaitForChild("Settings"):WaitForChild("SettingsHub")
 local isTenFootInterface = require(RobloxGui.Modules.TenFootInterface):IsEnabled()
+local HasVRAPI = pcall(function() return UserInputService.VREnabled and UserInputService.GetUserCFrame end)
```

Comment:
we are checking here for two things right? 1) if we have the api for vr and 2) if we have a vr device connected? Maybe I am remembering this wrong but isn't the first value just a bool whether the pcall succeeded?